packages/go/openapi/src/schemas/model.bh-graph.label.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+allOf:
+  - $ref: './model.bh-graph.font-icon.yaml'

```

Comment:
We probably don't want to make this object dependent on properties from `font-icon`. While they happen to share the same 3 properties here, we don't necessarily want `label` to change if `font-icon` does.

Reply:
Yea you're right, these really should be separate. fixed